kit/ForKit.cpp

```diff
@@ -437,6 +448,10 @@ static int createLibreOfficeKit(const std::string& childRoot,
 
             SigUtil::setSigChildHandler(nullptr);
 
+            // Throw away inherited watchdog, which will let a new one for this
+            // child be created on demand
+            SocketPoll::PollWatchdog.reset();
```

Comment:
Hmm - it might be nicer to just startThread() again in both child and parent - to save a potential race (which will never happen ;-) this should work just fine.